brave/brave-core
# Avoid excess classification text copying.

Avoid copying of string which size could be tens of KB or even more.


Resolves https://github.com/brave/brave-browser/issues/23329

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Verify that text classification resource is loaded
- Verify that text classification is working correctly (`Classified text with the` message on rewards-internals)
